app.py

```diff
@@ -353,6 +372,39 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
             f" worth of {int(example_row['liquidable_debt_at_interval']):,} USD will be liquidated while the AMM swaps "
             f"capacity will be {int(example_row['debt_token_supply']):,} USD."
         )
+  # accumulated liqudable debt table 
+    streamlit.header("Cumulative Liquidable Debt by Collateral Price")
+
+    # Create accumulated debt DataFrame
+    accumulated_debt = create_accumulated_debt(main_chart_data)
+
+    # Get min and max price once
+    price_min = float(accumulated_debt["collateral_token_price"].min())
+    price_max = float(accumulated_debt["collateral_token_price"].max())
+
+    # Create two columns for layout
+    col1, _ = streamlit.columns([1, 3])
+
+    # Slider for price range selection
+    with col1:
+        price_lower_bound, price_upper_bound = streamlit.slider(
+            label="Select range of Collateral Token Price",
+            min_value=price_min,
+            max_value=price_max,
+            value=(price_min, price_max),
+        )
+
+    # Filter accumulated_debt DataFrame based on selected price range
+    filtered_accumulated_debt = filter_by_price_range(
+        accumulated_debt, "collateral_token_price", price_lower_bound, price_upper_bound
+    )
+
+    # Display the filtered DataFrame and hide the index
+    streamlit.dataframe(
+        filtered_accumulated_debt[['collateral_token_price', 'liquidable_debt', 'Cumulative_liquidable_debt']],
```

Comment:
```suggestion
        main_chart_data[['collateral_token_price', 'liquidable_debt_at_interval', 'liquidable_debt']].round(),
```
This should actually do all the work 😅

Reply:
In fact, also pls rename `liquidable_debt` to `liquidable_debt_at_price`.